GOVERNANCE.rst

```diff
@@ -0,0 +1,138 @@
+.. _governance:
+
+WarpX Governance
+================
+
+WarpX is led in an open governance model, described in this file.
+
+
+Steering Committee
+------------------
+
+Current Roster
+^^^^^^^^^^^^^^
+
+- Jean-Luc Vay (chair)
+- Remi Lehe
+- Axel Huebl
+
+See: `GitHub team <https://github.com/orgs/ECP-WarpX/teams/warpx-admins>`__
+
+Role
+^^^^
+
+The steering committee (SC) can change repo requirements, can do administrative operations such as rename/move/archive projects, etc.
+SC members call votes for decisions.
+
+The SC can veto decisions of the technical committee (TC) by voting in the SC.
+
+The SC can change the governance structure, but only in an unanimous vote.
+
+Decision Process
+^^^^^^^^^^^^^^^^
+
+Decision are made in a public, majority vote of SC members.
```

Comment:
Do we need to specify how members will be notified of a vote (e.g. one week in advance email notice), and the voting procedure (e.g. Zoom meeting, or email)?

Reply:
Good questions, we tried to keep procedure slick for now but for the SC a bit more structure due to its weight might be useful.

```suggestion
Decision are made in a non-confidential, majority vote of SC members.
```